### PR TITLE
Add short markdown "how-to"

### PR DESCRIPTION
**Pull request by joepavitt**
Working with tracy, thought it would be beneficial to have a quick how-to guide on the most common Markdown formats

### CODE REVIEW

**Comment by knolleary** [comment]
Is it worth adding the raw markdown for that on the page as well? Otherwise you have to view the raw file to get the hints for the markdown syntax. I would have proposed the change but you can't easily propose changes that include "```" without mucking up the formatting.

File: development/markdown-how-to.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+navTitle: Markdown How-To
+---
+
+## Markdown Cheatlist
```
Comment (ZJvandeWeg) [inline development/markdown-how-to.md:5]: Can't we just link to the spec: https://daringfireball.net/projects/markdown/syntax ?

Comment (joepavitt) [reply development/markdown-how-to.md:5]: That's horribly formatted, and difficult to scan/parse for people new to the syntax

**Comment by joepavitt** [comment]
Add inline examples, and header separators for easier navigation/scan parsing

https://user-images.githubusercontent.com/99246719/220315907-795d67ee-84e0-47c1-9341-786795faca7a.mov

File: development/markdown-how-to.md (hunk 1 above)
Comment (knolleary) [reply development/markdown-how-to.md:5]: Whilst this is great for the basics to get started, we don't want to be maintaining a full page of docs on markdown. Including a link to a more complete reference would be helpful - and there are plenty out there that are more accessible than Gruber's page.

 - https://www.markdownguide.org/cheat-sheet/
 - https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

Comment (knolleary) [reply development/markdown-how-to.md:5]: To clarify - I suggest to include a link **in addition** to this high level cheat sheet of the basics.

**Comment by joepavitt** [comment]
> To clarify - I suggest to include a link in addition to this high level cheat sheet of the basics.

Actioned

**Review by knolleary — APPROVED** [review]

**Comment by joepavitt** [comment]
ignore that re-request @knolleary

**Comment by knolleary** [comment]
I got a bit of post-approval remorse, but not in time to block the merge. The paragraph example doesn't show a paragraph requires blank lines between them. Will PR quickly

**Pull request merged**